cms/utils/conf.py

```diff
@@ -67,9 +66,13 @@ def wrapper():
 
 
 def get_cache_durations():
-    return {
+    """
+    Returns the setting: CMS_CACHE_DURATIONS or will attempt to build it from
+    the deprecated settings or will return the defaults.
+    """
+    return getattr(settings, 'CMS_CACHE_DURATIONS', {
         'menus': getattr(settings, 'MENU_CACHE_DURATION', 60 * 60),
-        'content': get_cms_setting('CONTENT_CACHE_DURATION'),
+        'content': getattr(settings, 'CMS_CONTENT_CACHE_DURATION', 60),
```

Comment:
Because that setting is no longer supported. Doesn't matter though, I've removed it altogether based on @yakky's note.